sections/futureWork.tex

```diff
@@ -0,0 +1,7 @@
+Due to the promising results provided by the modified pachinko model, investigating this model further might be beneficial to incorporating metadata into topic models.
+However, testing these models on multiple datasets needs to be accounted for since the generalizability of these models is not explored within this paper.
+Using word embedding to further improve the performance of models can also be viewed as a possible next step for this project since a wide number of papers are using this technique to improve topic modeling\cite{MetaLDA2017}\cite{dieng2020topic}.
+
+It would also be interesting to see how we could incorporate these models into an existing IT infrastructure at Nordjyske.

```

Comment:
```suggestion
It would also be interesting to see how we could incorporate these models into an existing IT infrastructure for a news site such as Nordjyske.
```